app/javascript/redux/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import messagesreducer from './greetings/greetingsSlice';
```

Comment:
As suggested by the previous reviewer, there appears to be a mistake in spelling here as your file is named `greetingSlice.js` but you're importing `greetingsSlice.js` here. This is causing an error on the loading of the application. Please kindly fix and ensure your application is running without errors.:

<img width="688" alt="image" src="https://github.com/DianaBeki/hello-rails-react/assets/52934128/2e7c1f64-77d5-4cf9-a1aa-b439dd77d827">